AccessibilityCodelab/build.gradle

```diff
@@ -48,7 +53,7 @@ subprojects {
             targetExclude("$buildDir/**/*.kt")
             targetExclude('bin/**/*.kt')
 
-            ktlint("0.41.0")
+            ktlint("0.45.2")
```

Comment:
Missing userData

Reply:
this didn't seem to have user data before either, so i'm leaving it like this.